charts/alloy-lerian/values.yaml

```
@@ -0,0 +1,599 @@
+# it a single writer.
+#
+# Disable to reuse an existing instance — many client clusters already run one,
+# and running two is waste plus a potential write conflict.
+kube-state-metrics:
+  enabled: true
+  replicas: 1
+  # Automatic sharding is documented upstream as experimental and liable to
+  # break or be removed without notice. Our scale does not require it.
+  autosharding:
+    enabled: false
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)